app/src/test/resources/application-test.yml

```diff
@@ -49,5 +49,7 @@ bip:
   applicationName: "${BIP_APPLICATION_ISS}"
   applicationId: "${BIP_APPLICATION_ID}"
   stationId: "${BIP_STATION_ID}"
-truststore: "bipcert.jks"
-truststore_password: "vropassword"
\ No newline at end of file
+truststore: "${BIP_TRUSTSTORE:}"
+truststore_password: "${BIP_PASSWORD:}"
+keystore: "${BIP_KEYSTORE:}"
+bipalias: "${BIP_ALIAS:}"
```

Comment:
please finish the files with a empty line.

Reply:
Intellij removes it. I'll add it again.